glibc/README.md

```diff
@@ -0,0 +1,57 @@
+# Glibc base image
+
+This image compiles glibc for use with Alpine. 
+
+In order to build and push this image, you must build and push an image for each target architecture
+(`amd64` and `arm64`). Seeing as this is a compilation task, you must do this on a machine that uses
+the target architecture, push the images to Github, then create a consoildated manifest. This readme
```

Comment:
**question:** Is this strictly necessary, or is it just to avoid massive compile times for a non-native architecture?

Could we, for example, do `docker build . --platform=linux/arm64 -t ....` on an x86 machine to force an arm compilation?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org/) standard</sub>

Reply:
> is it just to avoid massive compile times for a non-native architecture?
Yes, that. I had to cancel the build after 3 hours on a non-native architecture.

Reply:
OK, yep, that sounds like a non-starter 😅